### DISCUSSION TITLE
Releases API (https://api.github.com/repos/OWNER/ORG/releases) is not listing all the releases

### DISCUSSION THREAD

**Discussion by billxinli** [community/community#41751] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

The releases API does not list all the releases (https://api.github.com/repos/OWNER/ORG/releases)

Sometimes a release will be visible on the GH list UI, but not appear in the rest call. 
Sometimes a release will not be visible on the GH list UI, however, if you visit the URL for the release directly, it exists.

When I recreate a "missing" release 422 is returned.

Edit:

I've paginated all the releases, only stops when zero records are returned.

**Comment by spenserblack** [comment]
Is it because it's paginated? Did you try `/repos/OWNER/REPO/releases?page=2`?

**Reply by billxinli** [reply]
Yep. I paginate the request with 100 items, and will loop until zero records are returned.

But it seems in the list view for releases is ordered by tags descending. Even in the list view within the UI itself, there are missing releases.
